### ISSUE TITLE
Crash when typing somthing in with the hud

### ISSUE THREAD

**Issue by ZavyFIre** [MeteorDevelopment/meteor-client#4477]
### Steps to reproduce

First make a server hud. Second make the server hud say {string(server)()}} and your game will crash

### Link to crash report/log (upload to https://mclo.gs and paste resulting link here)

https://mclo.gs/9yHCA4g

### Meteor Version

0.5.6

### Minecraft Version

1.20.4

### Operating System

Windows

### Before submitting a crash report

- [X] This crash wasn't already reported (I have searched crash reports on GitHub).

- [X] This is a valid crash (I am able to reproduce this on the latest dev build).

**Comment by GreemDev** [comment]
Yeah. Starscript failures aren't supposed to crash your game, but not putting in nonsense would help you not crash.

**Comment by ZavyFIre** [comment]
> Yeah. Starscript failures aren't supposed to crash your game, but not putting in nonsense would help you not crash.

I was trying to hide the last parts of the ip and I was just trying to do what made most sense in my mind.
